brainzutils/musicbrainz_db/event.py

```diff
@@ -0,0 +1,92 @@
+from collections import defaultdict
+from mbdata import models
+from brainzutils.musicbrainz_db import mb_session
+from brainzutils.musicbrainz_db.utils import get_entities_by_gids
+from brainzutils.musicbrainz_db.includes import check_includes
+from brainzutils.musicbrainz_db.serialize import serialize_events
+from brainzutils.musicbrainz_db.helpers import get_relationship_info
+
+
+def get_event_by_id(mbid):
+    """Get event with the MusicBrainz ID.
+
+    Args:
+        mbid (uuid): MBID(gid) of the event.
+    Returns:
+        Dictionary containing the event information.
+    """
+    event = _get_event_by_id(mbid)
+    return event
+
+
+def _get_event_by_id(mbid):
+    return fetch_multiple_events(
+        [mbid],
+        includes=['artist-rels', 'place-rels', 'series-rels', 'url-rels', 'release-group-rels'],
+    ).get(mbid)
+
+
+def fetch_multiple_events(mbids, *, includes=None):
```

Comment:
Keyword-only arguments is a feature for python3. It would be good to use keyword arguments since we would be using this module in Python2 projects as well.